src/components/PreviewComment/index.js

```diff
@@ -124,7 +124,7 @@ export const styles = StyleSheet.create({
   time: {
     fontFamily: fonts.inter[400],
     fontSize: 10,
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text color has been updated from `COLORS.blackgrey` to `COLORS.gray400`. This could potentially affect the readability of the text. Please ensure that the new color provides sufficient contrast against its background for optimal readability.
```diff
- color: COLORS.blackgrey,
+ color: COLORS.gray400,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->